vizro-core/examples/visual-vocabulary/pages/examples/waterfall.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+import pandas as pd
+import plotly.graph_objects as go
+import vizro.models as vm
+from vizro import Vizro
+from vizro.models.types import capture
+
+waterfall_data = pd.DataFrame(
+    {
+        "x": ["Sales", "Consulting", "Net revenue", "Purchases", "Other expenses", "Profit before tax"],
+        "y": [60, 80, 0, -40, -20, 0],
+    }
+)
+
+
+@capture("graph")
+def waterfall(
+    data_frame: pd.DataFrame,
+    x: str,
+    y: str,
+    measure: List[str],
+):
+    fig = go.Figure(
+        go.Waterfall(
+            x=data_frame[x],
+            y=data_frame[y],
+            measure=measure,
+            connector={"line": {"color": "rgb(63, 63, 63)"}},
```

Comment:
Don't worry about that now, but general design changes should live in the char template instead of being provided to the arguments. This way users can still overwrite the defaults if they use the function. For now, just remove it, and I'll take a look later what we should add to the template :) 

```suggestion
```

Reply:
Yes, makes sense!